refinery/core/models.py

```diff
@@ -1161,6 +1161,12 @@ class Meta:
         )
         ordering = ['-time_end', '-time_start']
 
+    @property
```

Comment:
What's the benefit of making this a property?

Reply:
No benefit, changing it now